src/main/java/net/dv8tion/jda/internal/managers/PermOverrideManagerImpl.java

```diff
@@ -92,7 +69,7 @@ private void setupValues()
     @Override
     public PermissionOverride getPermissionOverride()
     {
-        return override.resolve();
+        return override;
```

Comment:
This getter isn't following the "lookup `realEntity`" system that the other getters are following.
Is that intentional?

Reply:
Yes this is intentional because looking up the override is pretty complicated and I'd like to avoid that in this edge case.